### DISCUSSION TITLE
stressing super slicer with this STL

### DISCUSSION THREAD

**Discussion by legend069** [supermerill/SuperSlicer#1404] [General]
hello

so upon finding a slicer to slice this model for me i tried super slicer and well.. it slices it all weird like

decided to share the .STL here so it might be of use to help develop super slicer :)

[fan_outside - Copy.zip](https://github.com/supermerill/SuperSlicer/files/6795377/fan_outside.-.Copy.zip)

**Comment by supermerill** [comment]
I don't see what's weird 
![image](https://user-images.githubusercontent.com/6536403/125171393-4e5cce80-e1b4-11eb-9ed2-5bcb00bb31f4.png)

also, prefer sharing the .3mf (project save file) as it share the settings.

**Reply by legend069** [reply]
> 
> 
> I don't see what's weird
> ![image](https://user-images.githubusercontent.com/6536403/125171393-4e5cce80-e1b4-11eb-9ed2-5bcb00bb31f4.png)
> 
> also, prefer sharing the .3mf (project save file) as it share the settings.

i was just looking at how it sliced the first layer

![superslicer_4RAzzhXcHI](https://user-images.githubusercontent.com/40685552/125180924-09cc4400-e243-11eb-9906-538accb747ff.png)
[fan_outside_3mf.zip](https://github.com/supermerill/SuperSlicer/files/6796251/fan_outside_3mf.zip)
